libraries/bot-applicationinsights/src/test/java/com/microsoft/bot/applicationinsights/BotTelemetryClientTests.java

```diff
@@ -70,23 +70,21 @@ public void trackAvailabilityTest() {
         botTelemetryClient.trackAvailability(
             "test",
             OffsetDateTime.now(),
-            Duration.ofSeconds(1000), // TODO: use computer ticks
+            Duration.ofNanos(1000),
```

Comment:
Is this ok?

Reply:
C# [here](https://github.com/microsoft/botbuilder-dotnet/blob/main/tests/Microsoft.Bot.Builder.ApplicationInsights.Tests/BotTelemetryClientTests.cs#L73) use `TimeSpan(1000)`  and debugging it, I found that those 1000 are ticks but in Java there isn't something like that, I believe that nanos could be something similar. Apart from that, the `trackAvailability` method receives a Duration object, we cannot send it a Long or something to represent the ticks.

Reply:
Ack